contracts/Insurance.sol

```diff
@@ -68,17 +69,17 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
         updatePoolAmount();
         InsurancePoolToken poolToken = InsurancePoolToken(token);
 
-        // tokens to mint = (pool token supply / collateral holdings) * collaterael amount to stake
+        // tokens to mint = (pool token supply / collateral holdings) * collateral amount to stake
         uint256 tokensToMint =
             LibInsurance.calcMintAmount(
                 poolToken.totalSupply(),
-                collateralAmount,
+                publicCollateralAmount,
                 wadAmount
             );
 
         // mint pool tokens, hold collateral tokens
         poolToken.mint(msg.sender, tokensToMint);
-        collateralAmount = collateralAmount + wadAmount;
+        publicCollateralAmount = publicCollateralAmount + wadAmount;
```

Comment:
What is wadAmount? Is it the collateral that is being deposited into the pool when a user mints pool tokens?

Reply:
Yes, that's right